src/docs/product/performance/trace-view.mdx

```diff
@@ -0,0 +1,65 @@
+
+</Note>
+
+From Performance, Issues, and Discover, you can drill down into a single [trace](/product/performance/distributed-tracing) and traverse all of the transactions in that trace. This will accelerate your ability to debug slow services, identify related errors, and root out other bottlenecks.
+
+Currently, the trace view will only show up to 100 transactions from the trace. Additional transactions can be found through [Discover](/product/discover-queries).
+
+
+## Trace View
+
+The trace view is a split view where the left-hand side shows the trace's transaction tree, and the right-hand side represents each transaction as a colored rectangle. The first row listed is always the trace itself, from which all other spans in the trace descent.
+
+to find these views, you can either go through the [Transaction Summary](/product/performance/transaction-summary/), [Event Details](/product/performance/event-detail/), [Issue Details](/product/issues/issue-details/), or [Query Builder](/product/discover-queries/query-builder/). Trace IDs will be linked to the corresponding Trace Detail.
+
+## With Transaction Summary
+
+Select the [Performance Homepage](/product/performance/), then the affected transaction, and clicking on a trace ID in the table will display the trace data.
+
+## With Event Details
+
+Beneath the quick trace on the top right of the Event Details page, click on the trace ID will display the trace data.
+
+## With Issue Details
+
+Above the quick trace on the Issue Details page, click on View Full Trace will display the trace data.
+
+## With Query Builder
+
+Build a query with the trace column, then clicking a trace ID in the table will display the trace data.
+
+## Waterfall
+
+The waterfall view is a split view where the left reflects the trace's transaction tree, and the right reflects each transaction as a horizontal bar.
+
+## Transaction Details
+
+Clicking on a transaction row expands the details of that span. From here you can see many of the attached properties, such as tags and other field data. With the transaction name, you'll be able to view the [Transaction Summary](/product/performance/transaction-summary/) of that transaction across traces. Click on "View Summary" to see that summary.
+
+## Traversing Transactions
+
+Every transaction within a trace has additional details such as spans that. Click on "View Transaction" to see a detailed view of that transaction.
+
+# Troubleshooting
+
+## Orphan Traces and Broken Subtraces
+While in a fully instrumented system, a span in one service will give rise to a transaction in a subsequent service. There can be a variety of reasons why the transaction in the first service may not arrive in Sentry. When Sentry encounters transactions like this within a trace we link the transactions with a dashed line since they can no longer be directly connected to the root.
```

Comment:
```suggestion
In a fully instrumented system, a span in one service will connect to a transaction in a subsequent service. For a variety of reasons, a transaction in the first service may not arrive in Sentry. When Sentry encounters these types of transactions within a trace, we link the transactions with a dashed line since they can no longer be directly connected to the root.
```